samples/sampler/stories/qa/Panels.js

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable react/jsx-no-bind */
+import {handle, forward} from '@enact/core/handle';
+import kind from '@enact/core/kind';
+import {clamp} from '@enact/core/util';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {boolean, select} from '@enact/storybook-utils/addons/knobs';
+import Routable, {Route, Linkable} from '@enact/ui/Routable';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import Button from '@enact/agate/Button';
+import Header from '@enact/agate/Header';
+import Icon from '@enact/agate/Icon';
+import Item from '@enact/agate/Item';
+import {Panels, Panel, BreadcrumbPanels} from '@enact/agate/Panels';
+import Scroller from '@enact/agate/Scroller';
+
+import componentCss from './MenuItem.module.less';
+
+Panels.displayName = 'Panels';
+const Config = mergeComponentMetadata('Panels', Panels);
+
+const BasicPanels = () => {
+	const [index, setIndex] = React.useState(0);
+	const goNext = () => setIndex(clamp(0, 2, index + 1));
+	const goPrevious = () => setIndex(clamp(0, 2, index - 1));
+	const goFirst = () => setIndex(0);
+	const goThird = () => setIndex(2);
+
+	return (
+		<Panels
+			index={index}
+			noAnimation={boolean('noAnimation', Config, false)}
```

Comment:
Please check the order of Panel knobs. it should appear before the global knob. You will have to reposition the declaration.